Add Swedish translation

Bevinsky opened the pull request:
Here's a WIP of a Swedish translation.

I'm not all too comfortable with technical Swedish translation since I only use systems in English, so someone else should probably give this a look-over.

To show that your contribution is compatible with the MIT License, please include the following text somewhere in this PR description:  
This PR complies with the DCO; https://developercertificate.org/

Bevinsky committed:
Add Swedish TL.

Bevinsky committed:
Actually add the translation.

Bevinsky commented on copyparty/web/browser.js:
```diff
@@ -8185,6 +8185,635 @@ var Ls = {
+				"navigation",
+				["B", "växla mellan brödsmulor / trädvy"],
+				["I/K", "föregående/nästa mapp"],
+				["M", "hoppa till överordnad mapp (eller kollapsa närvarande)"],
```
"Överordnad mapp" sounds incredibly formal to me, but it's apparently what it's called.

chinatsu replied:
could probably change the phrasing to something like "jump to the directory above", but in swedish?

Bevinsky replied:
It could work if it's an action like here, but if it's in reference to just "parent folder" then it might sound weird... I'll have a look to see if it works in most contexts.

Bevinsky replied:
I think we'll just keep this for now unless someone really objects to it.

Bevinsky commented on copyparty/web/splash.js:
```diff
@@ -469,6 +469,47 @@ var Ls = {
+		"ad1": "detta stänger av all cachning; prova detta om k304 inte räckte till. Detta kommer att slösa enorma mängder nätverkstrafik!",
+		"ae1": "aktiva nedladdningar:",
+        "af1": "visa senaste uppladdningar",
+	},
```
I think I'm missing a few strings here, but I don't know where they are coming from.

icxes replied:
ag1 was missing from tl.js but should be there now
https://github.com/9001/copyparty/commit/c2cee222bdc3ae836ca1adb52f2af6381e8a5a1f#diff-759e097addf6e21177148e72a8b1763c84506b38e1614b99cade442eeaab4831

Bevinsky committed:
Add missing splash.js string.

Bevinsky committed:
Changed Okej to OK in dialogs.

Bevinsky committed:
Flip some words around.

Bevinsky committed:
Fix some typos and wordings.

Bevinsky committed:
Change some instances of permission names to Swedish.

It's a bit unclear if these should be preserved, since
the permissions aren't translated. But other translations
don't keep them, so...

Bevinsky commented on copyparty/web/browser.js:
```diff
@@ -8185,6 +8185,635 @@ var Ls = {
+		"ul_idle1": "inga uppladdningar har köats",
+		"ut_etah": "average &lt;em&gt;hashing&lt;/em&gt; speed, and estimated time until finish",
+		"ut_etau": "average &lt;em&gt;upload&lt;/em&gt; speed and estimated time until finish",
+		"ut_etat": "average &lt;em&gt;total&lt;/em&gt; speed and estimated time until finish",
```
Oops, I forgot these.

Bevinsky committed:
Add missing translations and change a word.

Bevinsky committed:
Fix string change from upstream.

Bevinsky committed:
Add new string.

Bevinsky committed:
Translated 'defer' which I forgot, and shortened another tl.

Bevinsky commented:
I would prefer if someone went through this, but there don't seem to be any other Swedes on the internet.

@9001 You can probably merge this unless there's something really egregious I've missed.

9001 commented:
najs! tack så mycket kompis :>
